pandas/tests/io/excel/__init__.py

```diff
@@ -1,6 +1,12 @@
 import pytest
 
-pytestmark = pytest.mark.filterwarnings(
-    # Looks like tree.getiterator is deprecated in favor of tree.iter
-    "ignore:This method will be removed in future versions:PendingDeprecationWarning"
-)
+pytestmark = [
```

Comment:
we need some method to eventually clean these up. open to ideas, can you create an issue.